enums/asset_level.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class AssetLevel(Enum):
```

Comment:
@Cuong-quoc-nguyen I think using enum in this case would make it easier to compare, can we rename the enum to match with business logic, otherwise, they should not be grouped that way then removing this enum is fine though.

Reply:
They should not be grouped in this way, they are not in the same data category on business-wide.

Reply:
I removed AssetLevel enum